test/export/test_export.py

```diff
@@ -2189,7 +2189,7 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
         em.module()(torch.randn(4, 3))
         with self.assertRaisesRegex(
             RuntimeError,
-            r"Runtime assertion failed for expression Eq\(Mod\(s0\*s1, s0 \- 1\), 0\)",
+            r"Runtime assertion failed for expression \(\(x\.size\(0\)\*x\.size\(1\)\) % \(x\.size\(0\) - 1\)\) == 0",
```

Comment:
Can you do better on parenthesization? Also, I would suggest converting this into assertExpectedInline tests, so it's easier to update the output strings when they wobble